.travis.yml

```diff
@@ -0,0 +1,46 @@
+language: c++
+compiler:
+  - clang
+  - gcc
+env:
+  - TRAVIS_NODE_VERSION="4"
+  - TRAVIS_NODE_VERSION="6"
+  - TRAVIS_NODE_VERSION="7"
+os:
+  - linux
+  - osx
+matrix:
+  fast_finish: true
+sudo: false
+cache:
+  directories:
+    - node_modules
+    - $HOME/.npm
+addons:
+  apt:
+    sources:
+      - ubuntu-toolchain-r-test
+    packages:
+      - g++-4.9
+before_install:
+  - echo "$TRAVIS_NODE_VERSION"
+  - export "PATH=./node_modules/.bin:$PWD/node_modules/.bin:$HOME/.local/bin:$PATH"
+  # coveralls
+  - pip install --user cpp-coveralls
+  # compilers
+  - if [ "$CXX" = "g++" -a "$TRAVIS_OS_NAME" = "linux" ]; then export CXX="g++-4.9" CC="gcc-4.9" AR="gcc-ar-4.9" RANLIB="gcc-ranlib-4.9" NM="gcc-nm-4.9" ; fi
+  - if [ "$CXX" = "clang++" ]; then export NPMOPT=--clang=1 ; fi
+  # node versions
+  - rm -rf ~/.nvm
+  - git clone --branch v0.33.2 https://github.com/creationix/nvm.git ~/.nvm
+  - source ~/.nvm/nvm.sh
+  - nvm install "$TRAVIS_NODE_VERSION"
+  - node --version
+  - export CFLAGS="$CFLAGS -O3 --coverage" LDFLAGS="$LDFLAGS --coverage"
+  - echo "CFLAGS=\"$CFLAGS\" LDFLAGS=\"$LDFLAGS\""
+install:
+  - npm install $NPMOPT
```

Comment:
This is not really needed, since after https://github.com/nodejs/node-addon-api/pull/44 there are no package dependencies. But I guess we could keep it in case there dependencies are added in the future?